### PR TITLE
Duplicate DRA EntityTransparentWhiteCircle

### PR DESCRIPTION
**Pull request by joshlory**
PSX: https://decomp.me/scratch/GW1Hi

Near-duplicate of RIC `EntityShrinkingPowerUpRing` with a few modifications.

### CODE REVIEW

File: src/dra/7E4BC.c [hunk 1]
```diff
@@ -892,7 +892,213 @@ void EntityHitByIce(Entity* self) {
...
+            prim1->u2 = prim1->u3 = 0x20;
+            prim1->v2 = prim1->v3 = 0xDF;
+            prim2->u2 = prim1->u0 = (prim2->u0 + prim1->u2) / 2;
+            prim2->v2 = prim1->v0 = (prim2->v0 + prim1->v2) / 2;
+            prim2->u3 = prim1->u1 = (prim2->u1 + prim1->u3) / 2;
+            prim2->v3 = prim1->v1 = (prim2->v1 + prim1->v3) / 2;
+            prim1->tpage = prim2->tpage = 0x1A;
+            prim1->clut = prim2->clut = 0x15F;
+            prim1->priority = prim2->priority = PLAYER.zPriority + 2;
+            prim1->drawMode = prim2->drawMode = 0x235;
```
Comment (sozud) [inline src/dra/7E4BC.c:951]: drawmode enum

**Review by sozud — APPROVED** [review]
Nice work

File: src/dra/7E4BC.c (hunk 1 above)
Comment (joshlory) [reply src/dra/7E4BC.c:951]: Whoops! you'd think I'd remember after https://github.com/Xeeynamo/sotn-decomp/pull/1379 😅

File: config/symbols.hd.dra.txt [hunk 2]
```diff
@@ -41,6 +41,7 @@ D_800ADC10 = 0x800ADCD4;
 D_800ADC44 = 0x800ADD08;
 D_800ADC7C = 0x800ADD40;
 D_800ADCC8 = 0x800ADD8C;
+D_800ADDE8 = 0x800ADEAC;
```
Comment (joshlory) [inline config/symbols.hd.dra.txt:44]: I'm not sure why this is needed when it's already been extracted to `src/dra/d_DBD4.c` below? but the build was [failing](https://github.com/Xeeynamo/sotn-decomp/actions/runs/9822606885/job/27119583447) without it...

Comment (sozud) [reply config/symbols.hd.dra.txt:44]: Doesn't look like it's extracted in the HD version https://github.com/Xeeynamo/sotn-decomp/blob/master/config/splat.hd.dra.yaml

Comment (joshlory) [reply config/symbols.hd.dra.txt:44]: Good catch

**Pull request merged**